src/main/java/eu/dissco/core/translator/terms/media/MediaType.java

```diff
@@ -15,13 +13,14 @@ public class MediaType extends Term {
 
   private final List<String> dwcaTerms = List.of(TERM, "dc:type");
   private final List<String> imageFormats = List.of("IMAGE/JPG", "JPG", "IMAGE/JPEG",
-      "JPEG", "IMAGE/PNG", "PNG");
+      "JPEG", "IMAGE/PNG", "PNG", "IMAGE/TIF", "TIF");
```

Comment:
Interesting, did not know TIF existed, I think it is good to discuss how we want to handle these different media types in the front-end (now it only displays plain images)

Reply:
Yes, would be good but I am not sure I will have an answer. Found that the DWCA that I tested with had TIF's in there so added them here